plugins/inputs/powerdns_recursor/sample.conf

```
@@ -8,3 +8,11 @@
   # socket_dir = "/var/run/"
   ## Socket permissions for the receive socket.
   # socket_mode = "0666"
+
+  ## The version of the powerdns control protocol to use. You will have to
+  ## change this based on your PowerDNS Recursor version, see below:
+  ## PowerDNS <4.5.0 => Protocol Version 1
+  ## PowerDNS 4.5.0 - 4.5.9 => Protocol Version 2
+  ## PowerDNS >=4.6.0 => Protocol Version 3
```

I think having these additional "=>" in there only creats confusion

```suggestion
  ## The version of the PowerDNS control protocol to use. You will have to
  ## change this based on your PowerDNS Recursor version, see below:
  ## Version 1: PowerDNS <4.5.0
  ## Version 2: PowerDNS 4.5.0 - 4.5.9
  ## Version 3: PowerDNS >=4.6.0
```